website/docs/reference/resource-configs/greenplum-configs.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Greenplum configurations"
+id: "greenplum-configs"
+---
+
+## Performance Optimizations
+    
+Table in greenplum have a few powerful optimizations to improve query perfomance:
+ 
+ - distribution
+ - column orientation
+ - compression
+ - appendonly toggle
+ - partitions
+ 
+Supplying these values as model-level configurations apply the corresponding settings in the generated `CREATE TABLE`(except partitions). Note that these settings will have no effect for models set to `view`.
```

Comment:
perhaps you could throw an error message if these configs are set for a view model?

Reply:
also, why does `partitions` not apply to a `CREATE TABLE` statement?

Reply:
yes, my bad, I mean `create table ... as select ...`

Reply:
"perhaps you could throw an error message if these configs are set for a view model?" I created an [issue](https://github.com/markporoshin/dbt-greenplum/issues/9).